Common/ov6piko.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+mkdir /tmp/kids
+mkdir /tmp/kids/logs
+pushd /tmp/kids
+
+# Download KIDS build for Plan 6 Lexicon
+curl -sSOL https://github.com/OSEHRA-Sandbox/VistA-M/releases/download/kcd7/UKO_KCD7_LOAD_0p2.KID
```

Comment:
always use latest release?

Reply:
This is kind of a special case. The kcd7 tag is probably not going to be changed, so we shouldn't use latest here.